app/views/admin/products/_account_fields.html.haml

```diff
@@ -1,5 +1,13 @@
 - if SettingsHelper.feature_on? :recharge_accounts
-  = f.input :facility_account_id, collection: current_facility.facility_accounts.active.collect {|fa| [ fa, fa.id ] }, hint: text("hints.deposit_account"), include_blank: false, label: FacilityAccount.model_name.human
+  -# include_blank: false is ignored if you use required: true
+  = f.input :facility_account_id,
+    label: FacilityAccount.model_name.human,
+    label_text: -> (label, _required, _) { "* #{label}" },
+    include_blank: false,
```

Comment:
Maybe my comment wasn't clear. If I added `required: true`, then it would ignore `include_blank: false`. The comment was meant to explain why I wasn't using `required: true`, when that's the more obvious choice (`label_text` is kind of an obscure simple_form method). If you have a suggestion for a better comment, I'm all for it.